package.json

```diff
@@ -5,13 +5,16 @@
   "main": "index.js",
   "type": "module",
   "scripts": {
-    "test": "echo \"Error: no tests are implemented yet \" && exit 1",
+    "test": "mocha **/*.test.js",
+    "test-watch": "nodemon --exec \"npm test\"",
     "coveralls": "cat ./coverage/lcov.info | ./node_modules/.bin/coveralls"
   },
   "author": "",
   "license": "MIT",
   "dependencies": {},
   "devDependencies": {
-    "coveralls": "^3.1.0"
```

Comment:
should nodemon be installed as a depency, so that the test-watch will work?

Reply:
I tried to run tests without test-watch and it works. Don't know if we need it

Reply:
it might be that you have nodemon installed globally into your machine. I tried with my own and it does not recognize 'nodemon' -command